components/form-builder/app/responses/Dialogs/ConfirmDialog.tsx

```diff
@@ -149,9 +149,12 @@ export const ConfirmDialog = ({
                     {t("downloadResponsesModals.confirmReceiptDialog.errors.invalidEntry.title")}
                   </Alert.Title>
                   <p>
-                    {t(
-                      "downloadResponsesModals.confirmReceiptDialog.errors.invalidEntry.description"
-                    )}
+                    <Trans
```

Comment:
I know, but I took it up a notch, and now you've got a recipe that lets you style your translation string with some basic HTML elements (italic in this case). As far as I've read and tested, the t function doesn't support it. Trans component does it but albeit not super intuitively https://react.i18next.com/latest/trans-component

Reply:
please let me know if you think it is a bad idea

Reply:
I'm not opposed to it ... to date we've broken strings apart which isn't a nice experience.

For sure it's not intuitive and the code without reading the docs looks like a mistake i.e. the empty ` <i></i>` 

Maybe leave a comment above the i tags to help any confusion on that end ...  just something helpful for the next dev to edit that code. 


Can we try using named tags?  Might be helpful for folks doing the translations (thoughts)

> Make sure you also adapt your translation resources to include the named tags (<italic>) instead of the indexed tags (<0>)!

Reply:
> import { Trans } from "next-i18next";

Re: the import + using ` Trans`

@bryan-robitaille any concerns thinking about the Next JS update + translations 👆

Reply:
I can't think of any conflicts at the moment.  In the Next JS update the i18n is initialized separately on the server and the client and the next-i18next package is removed.  We'll be using pure react-i18next and it 'should' work as long as the initialized t function is passed in.
I completely agree about using named tags as the indexes are not very straight forward and can quickly change when content is updated.